packages/vm/tests/api/state/stateManager.spec.ts

```diff
@@ -409,3 +409,80 @@ tape('StateManager - Contract code', (tester) => {
     t.end()
   })
 })
+
+tape('StateManager - Contract storage', (tester) => {
+  const it = tester.test
+
+  it('should throw on storage values larger than 32 bytes', async (t) => {
+    t.plan(1)
+    const stateManager = new DefaultStateManager()
+    const address = zeros(20)
+    const key = zeros(32)
+    const value = Buffer.from('aa'.repeat(33), 'hex')
+    try {
+      await stateManager.putContractStorage(address, key, value)
+      t.fail('did not throw')
+    } catch (e) {
+      t.pass('threw on trying to set storage values larger than 32 bytes')
+    }
+    t.end()
+  })
+
+  it('should strip storage values lower than 32 bytes', async (t) => {
```

Comment:
This test description is a bit misleading (stripping is logically not connected to the byte length) respectively also not correct (stripping is also done for 32-byte length. Can you please adjust?

Reply:
I have edited this name in 5217f50, hope this is better. I think this name was from when I still expected storage values to always be 32 bytes and I did not edit it right when I realized that was wrong.